elements/pl-number-input/pl-number-input.mustache

```diff
@@ -50,7 +64,27 @@
 {{#label}}<span>{{label}}</span>{{/label}}
 <samp>{{a_sub}}</samp>
 {{#suffix}}<span>{{suffix}}</span>{{/suffix}}
-{{#correct}}<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
+{{#correct}}
+<script>
+    $(function(){
+        $('#pl-number-input-correct-{{uuid}} [data-toggle="popover"]').popover({
+            container: 'body',
+            template: '<div class="popover pl-number-input-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+<span id="pl-number-input-correct-{{uuid}}">
+  {{^a_tru}}
+    <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>
+  {{/a_tru}}
+  {{#a_tru}}
+    <a class="btn btn-light border" role="button" data-toggle="popover" data-html="true" title="Correct Answer" data-content="The correct answer used for grading is <samp>{{a_tru}}</samp>" data-placement="auto" data-trigger="focus" tabindex="0">
```

Comment:
Can we change "The correct answer used for grading is" to "The exact correct answer is"? I think this would be clearer?

Reply:
I think this wording was chosen because sometimes multiple solution methods can work/will be within tolerance. That being said, I don't know if that is too specific to TAM 251.

Reply:
That's a fair point. I'm happy to leave the text as it is.